parser/serverless.yml

```diff
@@ -0,0 +1,34 @@
+service: auditlog
+
+provider:
+  name: aws
+  runtime: java8
+  stage: ${opt:stage, 'dev'}
+  region: eu-west-1
+  role: arn:aws:iam::500150530292:role/oph/omaopintopolkuloki/dev/lambda/parser/oma-opintopolku-loki-dev-parser-role
```

Comment:
Tämän vois siirtää ehkä funktio-tasolle? Ja hakee ehkä SSM Parameter storesta? 
https://serverless.com/framework/docs/providers/aws/guide/variables/#reference-variables-using-the-ssm-parameter-store